cockatrice/src/dlg_connect.cpp

```diff
@@ -124,12 +123,12 @@ DlgConnect::DlgConnect(QWidget *parent)
     QGroupBox *btnGroupBox = new QGroupBox(tr(""));
     btnGroupBox->setLayout(buttons);
 
-    QGridLayout *grid = new QGridLayout;
+    auto *grid = new QGridLayout;
```

Comment:
I don't really see the point in auto-fying these but I guess it doesn't hurt

Reply:
CLion says auto is better and more "modern" so I'm listening to it and fixing all the "warnings" i come across